homeassistant/components/googlehome/__init__.py

```diff
@@ -56,6 +58,11 @@
             discovery.async_load_platform(
                 hass, 'device_tracker', DOMAIN, device, config))
 
+        if device.get(CONF_TRACK_ALARMS):
```

Comment:
Since this has a default, you should use the [] syntax:
```suggestion
        if device[CONF_TRACK_ALARMS]:
```

Reply:
Roger that 👍